### PR TITLE
feat: adds opengraph to records

### PR DESCRIPTION
**Pull request by f-necas**
### Description

This PR introduces OpenGraph protocol to datahub's records. A nice implmentation would be to have a default image and base title for og:title.

But it will add more default.toml conf. I tried here to keep it simple.

🔗 ⚠️  Also add external viewer button to exports to be able to use it in other apps which use gn-ui

### Screenshots

image

### Quality Assurance Checklist

- [x] Commit history is devoid of any _merge commits_ and readable to facilitate reviews

**This work is sponsored by [MEL](https://lillemetropole.fr/)**.

### CODE REVIEW

File: apps/datahub/src/index.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
     <title>Datahub</title>
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:title" content="Datahub" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="" />
```
Comment (jahow) [inline apps/datahub/src/index.html:10]: Is this really required or will the angular code be enough to create those elements?

**Review by jahow — COMMENTED** [review]
Thank you, very interesting feature! Could you maybe put all the meta-related features in a component in the feature/record lib? something like `gn-ui-record-meta` for example. Thanks!

File: libs/feature/record/src/lib/record-meta/record-meta.component.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+} from '@angular/core'
+import { Meta } from '@angular/platform-browser'
+import {
+  DatasetRecord,
+  ServiceRecord,
+} from '@geonetwork-ui/common/domain/model/record'
+
+@Component({
+  selector: 'gn-ui-record-meta',
+  template: '<>',
```
Comment (jahow) [inline libs/feature/record/src/lib/record-meta/record-meta.component.ts:17]: it would be great if you could make this component standalone, thanks!

**Review by jahow — APPROVED** [review]
thank you! I just added a comment to remind you to make the new component standalone, thanks :)

File: libs/feature/record/src/lib/record-meta/record-meta.component.ts (hunk 2 above)
Comment (f-necas) [reply libs/feature/record/src/lib/record-meta/record-meta.component.ts]: Set as standalone and removed <> as it was adding this to html page :p

**Pull request merged**